modules/tracking/test/test_twist.cpp

```diff
@@ -0,0 +1,106 @@
+#include "test_precomp.hpp"
+
+#include "opencv2/core.hpp"
+#include "opencv2/tracking/twist.hpp"
+
+#define eps 1e-4
+
+namespace opencv_test
+{
+namespace
+{
+
+class TwistTest : public ::testing::Test
+{
+protected:
+    cv::detail::tracking::Twist twist;
+    cv::Mat K, J;
+
+    void SetUp() override
+    {
+        K = (cv::Mat_<float>(3, 3) << 1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0);
+    }
+};
+
+TEST_F(TwistTest, TestInteractionMatrix)
+{
+    // import machinevisiontoolbox as mv
+    // cam = mv.CentralCamera()
+    // print(cam.K)
+    // print(cam.visjac_p([1, 1], 2.0))
+    // [[1. 0. 0.]
+    //  [0. 1. 0.]
+    //  [0. 0. 1.]]
+    // [[-0.5  0.   0.5  1.  -2.   1. ]
+    //  [ 0.  -0.5  0.5  2.  -1.  -1. ]]
+
+    cv::Mat uv = (cv::Mat_<float>(2, 1) << 1.0, 1.0);
```

Comment:
Again, for such cases it's better to use `std::vector<float>` or `std::vector<cv::Vec3f>` or `cv::Mat` or similar and initialize it with { curly braced initializer list} and then convert it to `Mat`